src/backend/templates/rooms.hbs

```diff
@@ -592,6 +593,16 @@
             $('html, body').animate({scrollTop: 0}, 500);
         });
 
+        $('.date-button').click(function() {
+
+          let className = $(this).attr('class').split(' ')[0];
```

Comment:
Assuming the class will always be at position `0` is a bit fragile!

I think the idiomatic way would be to use a data attribute, instead of a class, when we want to attach simple data to an element.

```js
const className = $(this).attr('data-date');
```

In that case, the element might need to look more like this:
```handlebars
<a href="#" data-date="{{firstSlug}}" class="date-button tabs-nav-link">{{caption}}</a>
```

I might have got that wrong, but I hope you get the idea: `class=` for styling, `data-foo=` for data!

You can add as many `data-` attributes as you like. It is an open namespace for developers to use. (For example, Bootstrap uses things like `data-toggle` to add extra behaviour to an element.)